documentation/general/build-deb-rpm-locally.md

```diff
@@ -0,0 +1,29 @@
+# Build Deb or Rpm locally
+
+We do not need to build Debian or RPM locally too often. However, if we need, it is not productive to check in and wait for a build. Since the feedback loop could be 2 hours for a typo. The follow guide is a quick solution if you don't have an existing environment setup.
```

Comment:
nit: `...build since the feedback loop could be 2 hours for a typo. The following guide is a quick solution`